engine/lib/statistics.php

```diff
@@ -52,32 +52,47 @@ function get_entity_statistics($owner_guid = 0) {
 /**
  * Return the number of users registered in the system.
  *
- * @param bool $show_deactivated Count not enabled users?
+ * @param string $type count for the type - unverified, banned, total, active (default)
  *
  * @return int
  */
-function get_number_users($show_deactivated = false) {
-
-	$where = new \Elgg\Database\Clauses\EntityWhereClause();
-	$where->type_subtype_pairs = [
-		'user' => null,
-	];
-
-	if ($show_deactivated) {
-		$where->use_enabled_clause = false;
+function get_number_users($type = 'active') {
```

Comment:
we can not change the API functions in a breaking way (welcome to framework development)